weaver/main.go

```diff
@@ -49,6 +50,7 @@ func main() {
 	flag.BoolVar(&justVersion, "version", false, "print version and exit")
 	flag.StringVar(&ifaceName, "iface", "", "name of interface to read from")
 	flag.StringVar(&routerName, "name", "", "name of router (defaults to MAC)")
+	flag.StringVar(&nickName, "nickname", "", "nickname of host (defaults to container hostname)")
```

Comment:
You can't mention 'container' here; this code has no idea it's running as a container, and indeed might not.